ivy/functional/frontends/tensorflow/general_functions.py

```diff
@@ -448,3 +448,20 @@ def unstack(value: ivy.Array, axis=0, num=None, name=None):
 @to_ivy_arrays_and_back
 def reverse(tensor, axis, name=None):
     return ivy.flip(tensor, axis=axis)
+
+
+@to_ivy_arrays_and_back
+def approx_top_k(values, k=1, name=None):
```

Comment:
also, please match the parameters as per the original tf function, thanks!
ref: https://www.tensorflow.org/api_docs/python/tf/approx_top_k

Reply:
i will add a test function for this and make the requested changes